### DISCUSSION TITLE
[DEV] Help with making a repeatable quest

### DISCUSSION THREAD

**Discussion by nickolayis** [azerothcore/azerothcore-wotlk#3947] [General]
I need to make a quest repeatable infinite times just like the wool quest in stormwind but not for wool. Im on azerothcore  and cant seem to find any info on how to do that

**Comment by Kitzunu** [comment]
https://www.azerothcore.org/wiki/quest_template_addon#SpecialFlags

**Comment by nickolayis** [comment]
ty boss <3

**Comment by Helias** [comment]
I strongly recommend you to use this editor to make Quests, Items, Gameobjects, SmartAI etc.

[Keira3](https://github.com/azerothcore/Keira3/releases).

Soon, will be released a new version where you can see if a quest is repeatable or not by the quest-preview (similar to the item-preview).
